src/openai/_utils/_sync.py

```diff
@@ -1,56 +1,63 @@
 from __future__ import annotations
 
+import sys
+import asyncio
 import functools
-from typing import TypeVar, Callable, Awaitable
+import contextvars
+from typing import Any, Dict, List, TypeVar, Callable, Awaitable
 from typing_extensions import ParamSpec
 
-import anyio
-import anyio.to_thread
-
-from ._reflection import function_has_argument
-
 T_Retval = TypeVar("T_Retval")
 T_ParamSpec = ParamSpec("T_ParamSpec")
 
 
+
+if sys.version_info >= (3, 9):
+    to_thread: Callable[..., Awaitable[Any]] = asyncio.to_thread
```

Comment:
nit: I don't think you need the explicit annotation
```suggestion
    to_thread = asyncio.to_thread
```